src/HotChocolate/CodeGeneration/src/CodeGeneration.EntityFramework/Types/ForeignKeyDirective.cs

```diff
@@ -0,0 +1,35 @@
+using HotChocolate.Types;
+
+namespace HotChocolate.CodeGeneration.EntityFramework.Types
+{
+    public class ForeignKeyDirective
+    {
+        public string To { get; set; } = default!; // TODO: Potentially this supports nullable and can be derived by the field it annotates
```

Comment:
Once I reworked directives we could also have a constructor that passes this in.